docker-compose.yaml

```diff
@@ -163,7 +163,7 @@ services:
   postgres:
     image: postgres:16.3
     restart: always
-    container_name: postgres
+    container_name: postgres_spark
```

Comment:
I don't see any reason to do this - seems fine as is.

Reply:
so that we don't need to manually delete `postgres` container if we run docker-compose on this repo and on hive repo since they both have a `postgres` container.

Reply:
actually, I revert the container name changes. We need to update `POSTGRES_URL` for every other dependencies. It's not worth it.

Reply:
Plus I suspect we'll need to use the hive only compose file rarely at best.

Reply:
if we're going to change it we should change it there